src/assets/wise5/components/dialogGuidance/dialog-responses/dialog-responses.component.ts

```diff
@@ -18,4 +32,23 @@ export class DialogResponsesComponent {
   @Input() isWaitingForComputerResponse: boolean;
   @Input() responses: DialogResponse[] = [];
   @Input() showDetectedIdeas: boolean = false;
+
+  constructor(
+    protected dialog: MatDialog,
+    protected rubricEventService: RubricEventService
+  ) {}
+
+  protected openIdeasRubric(): void {
+    if (!this.rubricEventService.getIsRubricOpen()) {
+      this.dialog.open(CRaterRubricComponent, {
+        width: '40%',
+        position: { right: '0px', bottom: '0px' },
+        hasBackdrop: false,
+        data: {
+          cRaterRubric: this.cRaterRubric
+        }
```

Comment:
Since cRaterRubric is the only data that CRaterRubricComponent needs, just pass it without wrapping it into an object? Will need to change the constructor in CRaterRubricComponent too.
```suggestion
        data: this.cRaterRubric
        
```